src/game/PowerManager.cxx

```diff
@@ -180,4 +178,14 @@ void PowerManager::updatePowerLevels()
   {
     powerGrid.updatePowerLevel();
   }
+}
```

Comment:
These draft code is not related to the commit

Reply:
this is not draft code, I forgot to push my commit 🙈